packages/icons/src/classic-mode-icon.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import {
+  ClassicBike,
+  ClassicBus,
+  ClassicCar,
+  ClassicFerry,
+  ClassicGondola,
+  ClassicMicromobility,
+  ClassicTram,
+  ClassicWalk
+} from "./classic";
+
+/**
+ * Icons for all classic OTP-react-redux modes.
+ * Any hail and rental modes managed by one or multiple companies
+ * are optional but can be overriden here using the
+ * pattern <otp_mode>_<company_id> (e.g. 'car_hail_uber').
+ */
+function ClassicModeIcon({ mode, ...props }) {
+  if (!mode) return null;
+  switch (mode.toLowerCase()) {
+    case "bicycle":
+    case "bicycle_rent":
+      return <ClassicBike {...props} />;
+    case "bus":
+      return <ClassicBus {...props} />;
+    case "car":
+    case "car_park":
+      return <ClassicCar {...props} />;
+    case "ferry":
+      return <ClassicFerry {...props} />;
+    case "gondola":
+      return <ClassicGondola {...props} />;
+    case "micromobility":
+    case "micromobility_rent":
+      return <ClassicMicromobility {...props} />;
+    case "rail":
+    case "subway":
```

Comment:
I think it might be useful to add separate rail and subway icons at some point. This can be a separate issue for later though.

Reply:
I don't have a separate icon on hand, so that should be dealt with separately.

Reply:
Agreed.